node/src/serve.js

```diff
@@ -0,0 +1,19 @@
+import { readFileSync } from "fs"
+import dotenv from "dotenv"
+dotenv.config()
+
+const { API_PUBLIC_KEY, API_PRODUCT_TYPE } = process.env
+const validProductTypes = ["employment","income"]
+
+if(validProductTypes.indexOf(API_PRODUCT_TYPE) < 0) {
+  console.error("Not a Valid Product Type. Please specify an API_PRODUCT_TYPE of the following:")
+  console.error(validProductTypes)
+  process.exit(-1)
+}
+
+const html = readFileSync(`../html/${API_PRODUCT_TYPE}.html`).toString().replace("{{ public_key }}",API_PUBLIC_KEY).replace("{{ product_type }}",API_PRODUCT_TYPE)
```

Comment:
too long line, format it for readability

Reply:
this is now a multi-line statement. Looks better?

Reply:
yep thx